### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by ediowar** [egbertbouman/youtube-comment-downloader#36]
https://youtu.be/qFrbmseX9TI
When you try to download comments on archived livestreams, when you run the code in Python 2.7, an error occurs
```
Traceback (most recent call last):
  File "downloader.py", line 244, in <module>
    main(sys.argv[1:])
  File "downloader.py", line 239, in main
    print('Error:', str(e))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 28-38: ordinal not in range(128)
```

**Comment by egbertbouman** [comment]
Thanks for letting me know! I just fixed an issue for Python 2.7, but don't know if this issue is related. Could you check if this still happens with the latest version?

**Comment by ediowar** [comment]
This does not happen with the latest version.
